pkg/products/amqonline/prometheusRules.go

```diff
@@ -0,0 +1,166 @@
+package amqonline
+
+import (
+	"context"
+	"fmt"
+	monitoringv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/config"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+func (r *Reconciler) reconcileSloAlerts(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	keycloakServicePortCount := 2
+
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-slo",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{
+		{
+			Alert: fmt.Sprintf("AMQOnlineConsoleAvailable"),
```

Comment:
Is there a reason to use sprintf? when we are not building any string? Applicable in other places as well.

Reply:
@Rajagopalan-Ranganathan I've removed the use of `sprintf`. I'm not sure why is was there, but this is an existing alert that I simply moved into the newly created `prometheusRule.go` file, which is a file I've created for each product and contains all of the products related Prometheus rules.

The `reconcileKubeStateMetricsEndpointAvailableAlerts` and `reconcileKubeStateMetricsOperatorEndpointAvailableAlerts` functions within each products `prometheusRule.go` file are the new alerts that I have added.